frame/balances/src/lib.rs

```diff
@@ -222,7 +222,7 @@ pub mod pallet {
 		type ExistentialDeposit: Get<Self::Balance>;
 
 		/// The means of storing the balances of an account.
-		type AccountStore: StoredMap<Self::AccountId, AccountData<Self::Balance>>;
+		type AccountStore: StorageMapShim<Self::Account<Runtime>, frame_system::Provider<Runtime>, AccountId, Self::AccountData<Balance>>;
```

Comment:
you are not creating an example here, instead you are modifying the library.

An example would look like:
```rust
/// ```
/// construct_runtime!(...)
/// impl Balance for Runtime {... // here you would use storage map shim for the assocaited type account store
/// }
/// ...
/// ```

Reply:
Thanks @thiolliere, I will work on it.